projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -1135,6 +1137,11 @@ function ( $category_id ) {
 
 					break;
 
+				case 'in_site_migration_flow':
+					update_option( 'in_site_migration_flow', (int) (bool) $value );
```

Comment:
maybe it's a good idea to remove the option if `$value` is false? That way it will not be lingering in the site after migration.

Reply:
Done!